tools/mcetool.c

```diff
@@ -5719,6 +5747,21 @@ static const mce_opt_t options[] =
                         "set the delay for dumping core from unresponsive lipstick\n"
         },
         {
+                .name        = "set-input-policy-mode",
+                .with_arg    = xmce_set_input_policy_mode,
+                .values      = "enabled|disabled",
+                .usage       =
+                        "allow/deny grabbing of input devices based on input policy\n"
+                        "\n"
+                        "Normally this should be always set to 'enabled'.\n"
```

Comment:
what undesired effect may users experience if this flag is set to "enable" in a production-ready release?